internal/backend/rclone/config.go

```diff
@@ -0,0 +1,36 @@
+package rclone
+
+import (
+	"strings"
+
+	"github.com/restic/restic/internal/errors"
+	"github.com/restic/restic/internal/options"
+)
+
+// Config contains all configuration necessary to start rclone.
+type Config struct {
+	Program string `option:"program" help:"path to rclone (default: rclone)"`
+	Args    string `option:"args"    help:"arguments for running rclone (default: restic serve --stdio)"`
```

Comment:
That should probably be `(default: serve restic --stdio)`

Reply:
Yep